docker/Dockerfile

```
@@ -132,7 +132,7 @@ RUN --mount=type=cache,id=conda_pkgs,target=/opt/conda/pkgs,sharing=locked \
       python=${PYTHON_VER} && \
     # Clean and activate
     # conda clean -afy && \
-    sed -i 's/conda activate base/conda activate morpheus/g' ~/.bashrc
+    echo ". /opt/conda/etc/profile.d/conda.sh; conda activate morpheus" >> ~/.bashrc
```

```suggestion
    conda init &&\
    echo "conda activate morpheus" >> ~/.bashrc
```
Not sure the exact command but `conda init should handle sourcing the conda startup script for us.

---

`conda init bash` worked. thx